suite-native/module-accounts-management/src/components/TransactionListHeader.tsx

```diff
@@ -114,10 +122,25 @@ export const TransactionListHeader = memo(
             });
         };
 
+        const handleSend = () => {
+            navigation.navigate(RootStackRoutes.SendStack, {
+                screen: SendStackRoutes.SendOutputs,
+                params: {
+                    accountKey,
+                },
+            });
+        };
+
         const isTokenDetail = !!tokenContract;
         const isEthereumAccountDetail = !isTokenDetail && isEthereumAccountSymbol(account.symbol);
         const isPriceCardDisplayed = !isTestnetAccount && !isTokenDetail;
 
+        const isSendButtonDisplayed =
+            isSendEnabled &&
+            SEND_NETWORK_WHITELIST.includes(account.symbol) &&
+            !isTokenDetail &&
```

Comment:
isTokenDetail is probably not needed there

Reply:
I put it there intentionally. But you are right that when we enable ETH or other tokens supporting coin, we will definitely revisit this piece of code. So no need to handle it now. Removed in d4088165c7.